lib/src/main/java/com/thenewboston/bank/datasource/BankTransactionsDataSource.kt

```diff
@@ -0,0 +1,28 @@
+package com.thenewboston.bank.datasource
```

Comment:
Can we please try keep the package structure consistent with the endpoints? 

This shouldn't be in the bank endpoint package. It should be `banktransactions.datasource ` and same for anything class that talks to this endpoint.

Reply:
Ah okay, I will do the change

Reply:
Please verify